doc/conf.py

```diff
@@ -38,6 +38,7 @@
     'sphinx.ext.napoleon',
     'sphinx.ext.viewcode',
     "sphinx.ext.mathjax",
+    "sphinx.ext.todo",
```

Comment:
What's this?
Will it automatically generate TODO?

Reply:
This will support Todo section in docstring.
https://www.sphinx-doc.org/en/master/usage/extensions/todo.html
Without this extension sphinx raises warning for todo section.

Reply:
I see.
Thanks!